iorgen/types.py

```diff
@@ -409,6 +409,8 @@ def from_dict(cls: T[INPUT], dic: Dict[str, Any]) -> Optional[INPUT]:
             if "function_name" not in dic and "name" in dic:
                 print('WARNING: "name" is deprecated, use "function_name"')
                 dic["function_name"] = dic["name"]
+            if not re.fullmatch('[a-z][a-z0-9 ]*', dic['function_name']):
```

Comment:
You should add `A-Z` too

Reply:
Don't you specifically not want case-aware names so that it doesn't interfere with the camelCase naming?

Reply:
Well the name is lower cased anyway.
I guess we might want prevent the user to accidentally use a uppercase that will be lower cased. Or just probably issue a warning.
If we do that, let’s check the test samples, I think there is one or two of them that have uppercases.